pages/auth/policy.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import { useSession } from "next-auth/react";
+import { AcceptableUseTerms, AcceptableUseProps } from "@components/auth/AcceptableUse";
+import { requireAuthentication } from "@lib/auth";
+
+import { UserRole } from "@prisma/client";
+interface TermsOfUse {
+  content: string;
+}
+const TermsOfUse = (props: TermsOfUse) => {
+  const { data: session } = useSession();
```

Comment:
This call to useSession isn't necessary here because `getServerSideProps` is wrapped with `requireAuthentication`.
There is already a prop passed to this page called `user` that contains the `session.user` information.